modules/nw-udn-best-practices.adoc

```diff
@@ -15,6 +15,23 @@ Before setting up a `UserDefinedNetwork` (UDN) resource, users should consider t
 
 * `openshift-*` namespaces should not be used to set up a UDN.
```

Comment:
I think it should also say UDN should not be created in the `default` namespace.
Similar to the CUDN docs PR https://github.com/openshift/openshift-docs/pull/85165/files#diff-0a1ba53d7630d9df20cba09c7b2c755802671b1e1c7a170be621161e15af4fbcR13

Or at least warn that a UDN in default namespace would be accessible from all namespaces, result in no isolation that could introduce security risks for some use cases.

Reply:
Added:

* `UserDefinedNetwork` CRs should not be created in the default namespace. This can result in no isolation and, as a result, could introduce security risks to the cluster.